slash/react/src/Form/Slider/SliderInput.tsx

```diff
@@ -23,7 +23,12 @@ const SliderInput = ({
   ...sliderProps
 }: Props) => {
   const generatedId = useId();
-  const newId = useMemo(() => id ?? generatedId, [generatedId, id]);
+  const newId = useMemo(() => {
+    if (id && id !== "") {
```

Comment:
En faisant simplement ça il va prendre en compte la valeur null, undefined et empty string 
```suggestion
    if (id) {
```

Reply:
done